myia/infer/infnode.py

```diff
@@ -80,6 +80,18 @@ def _type_to_abstract(el):
     return type_to_abstract(el)
 
 
+def _without_value(abstract: data.AbstractAtom):
```

Comment:
Missing the decorator, also this is basically the `broaden` function that we had before.

```suggestion
@abstract_map.variant
def broaden(abstract: data.AbstractAtom):
```

Reply:
Ok!